src/daemon/rendezvous.rs

```diff
@@ -0,0 +1,73 @@
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
+
+use bytes::Bytes;
+use earendil_crypt::{Fingerprint, IdentityPublic, IdentitySecret};
+use earendil_packet::Dock;
+use moka::sync::Cache;
+use serde::{Deserialize, Serialize};
+use stdcode::StdcodeSerializeExt;
+
+pub const HAVEN_FORWARD_DOCK: Dock = 100002;
+
+pub struct ForwardTable {
```

Comment:
This does not need to be its own type, since its methods are straightforwardly accessing the two different cache. The entire interface here is that of two items, rather than implementing one coherent object.

You can see this by the fact that none of the methods touch more than 1 of the caches.